interleave.py

```diff
@@ -0,0 +1,11 @@
+from typing import Iterator
+
+
+
+def interleave(*args) -> Iterator[any]:
+    """
+      This function  get some args as of parameter and has to zip the lists
```

Comment:
1. Notice that there are double spaces between some of the words.
2. Please fix the grammer - for example: The function gets...